tests/test_core.py

```diff
@@ -8555,6 +8555,12 @@ def test_embind_lib_with_asyncify(self, args):
     self.emcc_args += args
     self.do_core_test('embind_lib_with_asyncify.cpp')
 
+  @no_asan('asyncify stack operations confuse asan')
+  def test_em_async_js(self):
+    self.uses_es6 = True
+    self.emcc_args += ['-s', 'ASYNCIFY', '--closure=1', '-Os']
```

Comment:
```suggestion
    self.set_setting('ASYNCIFY')
    self.maybe_closure()
```
(This will run closure when the test runner thinks it should, and also does not force `-Os` in all modes so we get more coverage.)

Reply:
Do we have test modes that do force `-Os`? I mainly wanted to make sure that test works in presence of mangling (since I temporarily introduced a bug there while adding this feature).

Reply:
I believe the -O2 mode will do mangling.

I believe the way to get closure coverage is to call `self.maybe_closure()`.

Reply:
Oops I see alon already said that part.

Reply:
Yes, we do test all name mangling in the various modes.

Reply:
Oh I missed the `maybe_closure` change in Alon's suggestion too.